src/store/domains/ui.js

```diff
@@ -0,0 +1,19 @@
+
+import { makeAutoObservable } from 'mobx';
+import i18n from '~/i18n';
```

Comment:
Um den UI-Store von `i18n` zu entkoppeln, könnte man überlegen, ob man die `i18n`-Funktion nicht ebenfalls als Konstruktorparameter in den Store mit reingibt. Evtl. müsste die `i18n`-Funktion etwas überarbeitet und dem Store vllt. mehr Kontrolle über die `i18n`-Möglichkeiten gegeben werden.